tests/sentry/rules/filters/test_issue_severity.py

```diff
@@ -0,0 +1,64 @@
+from sentry.issues.grouptype import GroupCategory
+from sentry.rules import MatchType
+from sentry.rules.filters.issue_severity import IssueSeverityFilter
+from sentry.testutils.cases import RuleTestCase
+from sentry.testutils.helpers.features import with_feature
+
+
+class IssueSeverityFilterTest(RuleTestCase):
+    rule_cls = IssueSeverityFilter
+
+    @with_feature("projects:first-event-severity-alerting")
+    def test_valid_input_values(self):
```

Comment:
Nit:
```suggestion
    def test_passing_input_values(self):
```

"Invalid" makes me think we're testing against strings or `None` or something.